app/templates/my_master.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'admin/base.html' %}
```

Comment:
Can we have a more meaningful name than `my_master`? Maybe like `admin_base` if that is the purpose of this file. Or else maybe just `header`